generated/nidcpower/nidcpower/_converters.py

```diff
@@ -275,3 +276,19 @@ def convert_chained_repeated_capability_to_parts(chained_repeated_capability):
     return [','.join(collections.OrderedDict.fromkeys(x)) for x in repeated_capability_lists]
 
 
+def convert_from_isolation_state_enum(enum):
+    if type(enum) is not enums._IsolationState:
+        raise TypeError('must be _IsolationState not {}'.format(type(enum).__name__))
+    return {
+        enums._IsolationState.ISOLATED: True,
+        enums._IsolationState.NON_ISOLATED: False,
+    }[enum]
+
+
+def convert_to_isolation_state_enum(value):
```

Comment:
If the user passes something that doesn't convert cleanly, you will get a KeyError. Probably should be a ValueError.

Reply:
With the try/except block in my latest commit, the converter function will now through a ValueError instead of KeyError if the user input value cannot be converted.